pages/common/dotnet-test.md

```diff
@@ -0,0 +1,16 @@
+# dotnet test
+
+> Execute tests for a .NET application.
```

Comment:
```suggestion
> Execute tests for a .NET application.
> Note: View <https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests#syntax> for supported filter expressions.
```

Reply:
Okay, I believe on the third line after the `> More information (...)` line would be good.